Backend/Converter/ChainsawToLogConverter.cs

```diff
@@ -78,7 +76,11 @@ public class ChainsawToLogConverter : ILogConverter {
                                 continue;
                             }
                             if (child.Name.EndsWith("message")) {
-                                log.Message = child.InnerText;
+                                log.Message += child.InnerText;
+                            }
+                            if (child.Name.EndsWith("NDC"))
```

Comment:
Could you provide a configuration example for log4j so we can test this and extend the example in the comment (see above) so it's documented?